src/lib/UIKitConfigProvider/utils/__tests__/getConfigsByPriority.spec.ts

```diff
@@ -0,0 +1,80 @@
+import getConfigsByPriority from '../getConfigsByPriority';
+
+const mockRemoteConfigs = {
+  common: {
+    enableUsingDefaultUserProfile: false,
+  },
+  groupChannel: {
+    channel: {
+      enableMention: false,
+      enableOgtag: true,
+      enableReactions: true,
+      enableTypingIndicator: true,
+      enableVoiceMessage: false,
+      input: {
+        camera: {
+          enablePhoto: true,
+          enableVideo: true,
+        },
+        enableDocument: true,
+        gallery: {
+          enablePhoto: true,
+          enableVideo: true,
+        },
+      },
+      replyType: 'quote_reply',
+      threadReplySelectType: 'thread',
+    },
+    channelList: {
+      enableMessageReceiptStatus: false,
+      enableTypingIndicator: false,
+    },
+    setting: {
+      enableMessageSearch: false,
+    },
+  },
+  openChannel: {
+    channel: {
+      enableOgtag: true,
+      input: {
+        camera: {
+          enablePhoto: true,
+          enableVideo: true,
+        },
+        enableDocument: true,
+        gallery: {
+          enablePhoto: true,
+          enableVideo: true,
+        },
+      },
+    },
+  },
+};
+
+describe('getConfigsByPriority', () => {
+  it('should override remoteConfigs value \w localConfigs which has higher priority', () => {
+    const localConfigs = {
+      common: {
+        enableUsingDefaultUserProfile: true,
+      },
+    };
+    const remoteConfigs = {
```

Comment:
nit
```suggestion
    const remoteConfigs: UIKitConfigInfo = {
```

Reply:
Addressed in https://github.com/sendbird/sendbird-uikit-react/pull/569/commits/ab6deccbd62ba1d49222a0a64953277dfa3287f4